pubpub/platform
# feat: add react-email for creating pretty emails

## Issue(s) Resolved

This adds the email rendering framework `react-email`, which allows you to write email templates with react and tailwind. really nice!

See https://react.email/ for more info

## Test Plan

1. `pnpm i`
2. `pnpm dev`
3. Request a password reset email
4. See the glory
5. Go to `localhost:3003`
6. Edit something in `packages/emails/src/password-reset.tsx`
7. See live reloading of the template! very cool

## Screenshots (if applicable)

Get email like this:

<img width="1000" alt="image" src="https://github.com/user-attachments/assets/63a18b9e-ecc2-4c54-9aae-e9337232faaf">


Instead of this 🤮:

<img width="858" alt="image" src="https://github.com/user-attachments/assets/47dbcc45-275e-4129-a2b4-b4112ac7cc1d">



## Optional

### Notes/Context/Gotchas

### Supporting Docs
